Disable tests that use dbus because we do not want to declare a
dependency on something only used to run optional tests and we do not
want the test suite to find and use dbus opportunistically which could
fail if glib2 is being built universal and dbus is not installed
universal.
--- gio/tests/meson.build.orig	2020-10-01 07:43:53.000000000 -0500
+++ gio/tests/meson.build	2021-12-15 18:20:06.000000000 -0600
@@ -99,8 +99,7 @@
 
 # Check for libdbus1 - Optional - is only used in the GDBus test cases
 # 1.2.14 required for dbus_message_set_serial
-dbus1_dep = dependency('dbus-1', required : false, version : '>= 1.2.14')
-if not dbus1_dep.found()
+if false
   if cc.get_id() == 'msvc' or cc.get_id() == 'clang-cl'
     # MSVC: Search for the DBus library by the configuration, which corresponds
     # to the output of CMake builds of DBus.  Note that debugoptimized
@@ -112,7 +111,7 @@
     endif
   endif
 endif
-if dbus1_dep.found()
+if false
   glib_conf.set('HAVE_DBUS1', 1)
 
   gio_tests += {
@@ -202,7 +201,7 @@
   endif
 
   #  Test programs that need to bring up a session bus (requires dbus-daemon)
-  have_dbus_daemon = find_program('dbus-daemon', required : false).found()
+  have_dbus_daemon = false
   if have_dbus_daemon
     annotate_args = [
       '--annotate', 'org.project.Bar', 'Key1', 'Value1',
